src/sentry/static/sentry/app/components/events/eventEntries.jsx

```diff
@@ -242,6 +244,15 @@ class EventEntries extends React.Component {
         {!isShare && event.sdkUpdates && event.sdkUpdates.length > 0 && (
           <EventSdkUpdates event={event} />
         )}
+        {!isIssuesPage && (
```

Comment:
What about we move the related events above the SDK info?
Given that SDK info is more useful to troubleshoot Sentry SDK related issues

Reply:
the problem is that the related events component hasn't the same pattern as the SDK's... it will look a bit weird.. Let me clarify this today with Dora 😉